src/api/comments.js

```diff
@@ -0,0 +1,26 @@
+import { getData } from './api';
+
+const BASE_URL = 'https://mate-api.herokuapp.com';
```

Comment:
import BASE_URL from api.js

```suggestion
import { getData, BASE_URL } from './api';
```